doc/architecture/decisions/0002-use-elastic-co-docker-image.md

```diff
@@ -0,0 +1,23 @@
+# 2. Use Docker Image from elastic.co
+
+Date: 2017-06-15
+
+## Status
+
+Accepted
+
+## Context
+
+[ElasticSearch](https://www.elastic.co/products/elasticsearch) is a Java based search engine built on top of technologies like Lucene and Solr. It is open source software which can be freely [downloaded](https://github.com/elastic/elasticsearch) and installed on a number of operating systems. It is also available from elastic.co as both a Docker image and as a cloud service.
+
+## Decision
+
+For initial proof of concept and early beta the Docker image is the quickest way of getting up and running with ES and this outweighs the concerns about the inability to change the default user/password for the user which ES runs as.
```

Comment:
The latter part of this sentence seems more like a consequence to me. Perhaps rewrite to `We will use the official Docker image provided by elastic.co`. I don't know if the bit about initial POCs and early Beta is required, especially as these things tend to no just be for that initial period.